### PR TITLE
Several syntax amendments and tiny improvements

### PR DESCRIPTION
**Pull request by Zhang-Junzhi**
***
<a href="https://pr-preview.s3.amazonaws.com/Zhang-Junzhi/selection-api/pull/106.html" title="Last updated on May 23, 2019, 6:58 AM UTC (da7e150)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/selection-api/106/ed19ac5...Zhang-Junzhi:da7e150.html" title="Last updated on May 23, 2019, 6:58 AM UTC (da7e150)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -422,10 +423,10 @@ <h2>Selection interface</h2>
 
         <dt><dfn>deleteFromDocument</dfn></dt>
         <dd>
-          <p>The method must invoke <dfn><a href="https://dom.spec.whatwg.org/#dom-range-deletecontents"><code>deleteContents()</code></a></dfn> ([[!DOM]])
+          <p>The method must invoke <dfn><a href="!DOM#dom-range-deletecontents"><code>deleteContents()</code></a></dfn> ([[!DOM]])
```
Comment (marcoscaceres) [inline index.html]: Should be data-cite :) ... don't need ! anymore... will update those in a different PR. 
```suggestion
          <p>The method must invoke <dfn><a data-cite="DOM#dom-range-deletecontents"><code>deleteContents()</code></a></dfn> ([[DOM]])
```

File: index.html [hunk 2]
```diff
@@ -591,9 +593,9 @@ <h3><code><dfn>selectstart</dfn></code> event</h3>
 
         <p>When the user agent is about to associate a new range <var>newRange</var> to the <a>selection</a> in response to a user initiated action,
           the user agent must <dfn><a href="https://www.w3.org/TR/dom/#concept-event-fire">fire</a></dfn> ([[!DOM]])
-          an event with the name <code>selectstart</code>, which bubbles and is cancelable</a>,
-          at the <a>node</a> associated with the <a>boundary point</a> of <var>newRange</var>'s <a>start</a>
-          prior to changing the selection if the <a>selection</a> was previously <a>empty</a>
+          an event with the name <code>selectstart</code>, which <a href="!DOM#dom-event-bubbles">bubbles</a> and is
+          <a href="!DOM#dom-event-cancelable">cancelable</a>, at the <a>node</a> associated with the <a>boundary point</a> of <var>newRange</var>'s
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          <a data-cite="DOM#dom-event-cancelable">cancelable</a>, at the <a>node</a> associated with the <a>boundary point</a> of <var>newRange</var>'s
```

File: index.html [hunk 3]
```diff
@@ -607,8 +609,8 @@ <h3><code><dfn>selectionchange</dfn></code> event</h3>
         <p>When the <a>selection</a> is dissociated with its <a>range</a>, associated with a new <a>range</a>
           or the associated <a>range</a>'s <a>boundary point</a> is mutated either by the user or the content script,
           the user agent must <dfn><a href="https://www.w3.org/html/wg/drafts/html/master/webappapis.html#queue-a-task">queue a task</a></dfn>
-          to <a>fire</a> an event with the name <code>selectionchange</code>, which does not bubble and is not cancelable</a>,
-          at the <a>document</a> associated with the <a>selection</a>.</p>
+          to <a>fire</a> an event with the name <code>selectionchange</code>, which does not <a href="!DOM#dom-event-bubbles">bubble</a> and is not
+          <a href="!DOM#dom-event-cancelable">cancelable</a>, at the <a>document</a> associated with the <a>selection</a>.</p>
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
          <a data-cite="DOM#dom-event-cancelable">cancelable</a>, at the <a>document</a> associated with the <a>selection</a>.</p>
```

File: index.html [hunk 4]
```diff
@@ -618,13 +620,14 @@ <h2>Acknowledgements</h2>
       <p>Many thanks to</p>
       <ul>
         <li><p>Aryeh Gregor, who is the original author of this specification as well as <a href="https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html">HTML Editing API specification</a>.</p></li>
-        <li><p>Contributors to the <a href="https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html">HTML Editing API specification</a> - 
+        <li><p>Contributors to the <a href="https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html">HTML Editing API specification</a> -
```
Comment (marcoscaceres) [inline index.html]:
```suggestion
        <li>Contributors to the <a href="https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html">HTML Editing API specification</a> -
```

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
Some suggestions to quickly apply

**Comment by Zhang-Junzhi** [comment]
Thanks for the review. I made a force-push to modify them.

**Review by marcoscaceres — APPROVED** [review]

**Comment by marcoscaceres** [comment]
Marked as non substantive for IPR from ash-nazg.

**Pull request merged**